f/FPC/FPC-3.2.2.eb

```diff
@@ -0,0 +1,22 @@
+name = 'FPC'
+version = '3.2.2'
+
+homepage = 'https://www.freepascal.org/'
+description = """Free Pascal is a mature, versatile, open source Pascal compiler. It can target many processor architectures"""
+
+toolchain = SYSTEM
+
+source_urls = ['http://downloads.freepascal.org/fpc/dist/3.2.2/i386-linux']
+sources = ['fpc-3.2.2.i386-linux.tar']
```

Comment:
Not sure to what extent it matters, but this is for a 32-bit architectures, right?
I rather expected to see the 64-bit one here (x86_64).

Reply:
Well spotted, this was a mistake on my side ([063396f](https://github.com/vscentrum/vsc-software-stack/pull/229/commits/063396f9cc9841cfeb0c95a4d5c08d956006394a))